src/content/changes/highlights/2023-02-16.md

```diff
@@ -9,7 +9,12 @@ owner:
 
 
 ## User Interfaces
-
+- [coredns-app](https://github.com/giantswarm/coredns-app) [v1.14.2](https://github.com/giantswarm/coredns-app/releases/tag/v1.14.2) changes the Pod Disruption Budget from `maxUnavailable: 1` to `maxUnavailable: 25%` for better scaling. Drop static `replicas`, managed by HPA from the Deployment and add changes to the Config Map by adding 5 seconds to the Health Check
+- [k8s-dns-node-cache-app](https://github.com/giantswarm/k8s-dns-node-cache-app) [v2.0.0](https://github.com/giantswarm/k8s-dns-node-cache-app/releases/tag/v2.0.0) introduces support for Cilium CNI in EBPf mode and should be **only** used with AWS v19 release
```

Comment:
```suggestion
- [k8s-dns-node-cache-app](https://github.com/giantswarm/k8s-dns-node-cache-app) [v2.0.0](https://github.com/giantswarm/k8s-dns-node-cache-app/releases/tag/v2.0.0) has been released. It introduces a new major version in order to support Cilium CNI in EBPf mode. The new major version should be **only** used from AWS v19 release and so on.
```

Reply:
@pipo02mix I try to commit your suggestion but it does not work :(

Reply:
done. I had to copy and paste as suggestion again 🤷

Reply:
@T-Kukawka are you fine with the changes made by @pipo02mix as I just copied your notes from two weeks ago?